mcu-util/src/orb/revision.rs

```diff
@@ -14,15 +14,33 @@ impl Display for OrbRevision {
         } else if self.0.version
             < orb_messages::hardware::OrbVersion::HwVersionDiamondPoc1 as i32
         {
-            write!(f, "EVT{:?}", self.0.version)
+            // Pearl
+            if self.0.version
+                <= orb_messages::hardware::OrbVersion::HwVersionPearlEv5 as i32
+            {
+                write!(f, "EVT{:?}", self.0.version)
```

Comment:
```suggestion
                write!(f, "Pearl EV{:?}", self.0.version)
```
I'd prefer verbose unless any tools depends on it

Reply:
i would prefer too but indeed, this would break things